test/unit/index.html

```diff
@@ -14,7 +14,7 @@
     <meta charset="utf-8">
     <title>Tests</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <script src="../../node_modules/wct-browser-legacy/browser.js"></script>
```

Comment:
You're going to need to add `<script src="../../node_modules/mocha/mocha.js"></script>` here before wct-mocha because wct-mocha needs mocha to exist.